extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -246,11 +247,11 @@ describes.realWin('amp-story', {
     createPages(story.element, 2, ['cover', 'page-1']);
     return story.layoutCallback()
         .then(() => {
-        // Getting all the AmpStoryPage objects.
+          // Getting all the AmpStoryPage objects.
           const pageElements =
             story.element.getElementsByTagName('amp-story-page');
           const pages = Array.from(pageElements).map(el => el.getImpl());
-          return Promise.all(pages);
```

Comment:
I deleted this return because it seems it wouldn't let the following code run, but let me know if I'm missing something.

Reply:
By itself, `Promise.all(pages)` won't really do anything.  If you want to wait for the promises to resolve (which, maybe we do?), then you'll want `Promise.all(pages).then(() => { ... })`

Reply:
Actually we didn't even need that line and the test wasn't running correctly, I fixed it in the last commit.